json/testnet-bitcoin-config.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
I feel we should have separate config files used specifically for docker, because these are still useful for directly, non-containerized installs. Maybe just create them in the `docker/` folder?

Reply:
Potentially, but unless people are careful remembering there are two places to update the value, this problem occurs.

Testnet config was updated [here](https://github.com/decentralized-identity/ion/blob/master/json/testnet-bitcoin-config.json#L9) with the latest release and was not updated [here](https://github.com/decentralized-identity/ion/blob/master/docker/config.sh#L74) meaning the docker compose script does not run at the moment as reported in #99 and #100.

Reply:
I'd prefer we dont have two authoritative places for the same value.